src/sdl2/mixer/mod.rs

```diff
@@ -790,6 +790,28 @@ impl<'a> Music<'a> {
         }
     }
 
+    /// Load music from a byte buffer.
+    pub fn from_bytes(buf: &[u8]) -> Result<Music<'static>, String> {
```

Comment:
This is pretty unsafe, if the `&[u8]` comes from a Vec that gets destroyed right afterwards, this will probably crash, unless I'm missing something.

Please replace that by `&'static [u8]`, and change the fn name to `from_static_bytes`.

Reply:
You're right; I tried that and it crashed immediately.